src/server/pages.rs

```diff
@@ -68,6 +68,13 @@ pub struct SdpFileRequest {
     source: String,
 }
 
+#[derive(Apiv2Schema, Debug, Deserialize)]
+pub struct ThumbnailFileRequest {
+    source: String,
+    quality: Option<u8>,
```

Comment:
What quality is not clear, can you add a comment ?

Reply:
You should use a validator here for the range.
https://github.com/Keats/validator

Reply:
Done, after a fight with versioning